src/app/drive/components/FileViewer/FileViewer.tsx

```diff
@@ -89,6 +95,12 @@ const FileViewer = ({
   const isEditNameDialogOpen = useAppSelector((state: RootState) => state.ui.isEditFolderNameDialog);
   const isShareItemSettingsDialogOpen = useAppSelector((state) => state.ui.isShareItemDialogOpenInPreviewView);
 
+  const trackFilePreviewProperties: TrackingPlan.FilePreviewProperties = {
+    file_size: file.size,
+    file_extension: file.type,
+    preview_id: file.uuid,
```

Comment:
This is crashing public shared file view.

Reply:
Fixed